dts/arm/silabs/efr32fg1p133f256gm48.dtsi

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2018 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_SOC_EFR32FG1P133F256GM48
```

Comment:
If we're to create additional dtsi files for SoC variants, I think the proposal in the  first commit in cleaner